Web GUI testing

artbez opened the pull request:
To solve #36 issue.
These tests use Selenium library. It setups chrome browser which simulates user actions. 
Server must have Chrome browser installed for running tests

artbez committed:
Add a module for gui-testing and simple test
1. Add module for gui-testing
2. Add test with auth by login "123" and password "123" in order to open dashboard page
3. Add test with auth by random login and random password in oder to see error on the page

artbez committed:
Retain userWrongAuth

1. Make userWrongAuth test working correctly
2. Add wait to userCanLoginByUsername so that dashboard page managed to load

artbez committed:
Add travis support for GUI testing

1. Add .property file for UI-testing in oder to specify profiles
2. Modify .travis.yml file so that it includes GUI testing

artbez committed:
Add JavaDoc and fix checkstyle's bugs

jzuken commented on .travis.yml:
```diff
@@ -6,6 +6,9 @@ os:
 dist: trusty
 
+addons:
+  firefox: "50.0"
```
do we still need firefox? I thought now we use chrome

jzuken commented on UI-testing/UI-testing.iml:
```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
I think we should rename it into lowercase for uniformity

jzuken commented:
we need a wiki page with instructions on how to setup all gui testing environment on developers' computers

artbez committed:
Rename UI-testing

1. Rename UI-testing to ui-testing
2. Remove firefox addon from travis

artbez committed:
Link callTomcat.sh with ui-testing

Add possibility to run callTomcat.sh from gui-tests in oder to check is tomcat setuped or not
